radio/src/gui/212x64/model_setup.cpp

```diff
@@ -1122,9 +1166,12 @@ void menuModelSetup(event_t event)
 #endif
       if (IS_MODULE_R9M(moduleIdx)) {
         lcdDrawTextAlignedLeft(y, TR_MULTI_RFPOWER);
-        lcdDrawText(MODEL_SETUP_2ND_COLUMN, y, getR9MPowerString(g_model.moduleData[moduleIdx].pxx.power), attr);
+        if (IS_MODULE_R9M_FCC(moduleIdx))
+          lcdDrawTextAtIndex(MODEL_SETUP_2ND_COLUMN, y, STR_R9M_FCC_POWER_VALUES, g_model.moduleData[moduleIdx].pxx.power, LEFT | attr);
+        else
+          lcdDrawTextAtIndex(MODEL_SETUP_2ND_COLUMN, y, STR_R9M_LBT_POWER_VALUES, g_model.moduleData[moduleIdx].pxx.power, LEFT | attr);
```

Comment:
This doesn't actually show up in any of the UI sizes.  I don't think it's supposed to (since power is determined by telemetry on/off), but I just thought I'd mention it.  Maybe a comment could be left in these spots (or code removed?).

Reply:
good catch